Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Nabla"
 uuid = "49c96f43-aa6d-5a04-a506-44c7070ebe78"
-version = "0.12.1"
```

Comment:
dropping Julia 1.0 is not breaking 
https://github.com/SciML/ColPrac#dropping-support-for-earlier-versions-of-julia

although ideally it'd be a minor release, so we could back-port things... but we're pre-1.0.0 so we don't have that privilege

Reply:
ohh ok. I assumed it was breaking since it completely cuts off versions of julia. my bad

Reply:
If we're wrong about this the General registry checks will let us know